### PR TITLE
683 : Adding documentation changes

### PR DESCRIPTION
**Pull request by Saket2**

### CODE REVIEW

File: core-concepts/building-the-ui/README.md [hunk 1]
```diff
@@ -32,16 +32,16 @@ Appsmith has a collection of widgets that can be used to build the UI. Widgets c
...
 
 ## Editing widget properties 
 
-Widget properties can be editing via the property pane which can be toggled next to the delete icon. Widget properties can affect data, style and action to take. 
+Widget properties can be edited via the property pane which could be opened using top-right icon (Edit Widget Properties). Widget properties can affect the widget data, styling and actions.
 
 ![](../../.gitbook/assets/input-property-pane.png)
 
-**Widget properties don't have to be static!** Connect a widget property to an API by typing **`{{ }}`** inside the property and binding the property using an API name or Widget name. You can learn more about working with widgets and APIs here
+**Widget properties don't have to be static!** To make it dynamic, type **`{{ }}`** inside the widget property and bind it to an API name or Widget name. You can learn more about working with widgets and APIs here
```
Comment (Nikhil-Nandagopal) [inline core-concepts/building-the-ui/README.md:44]: @Saket2 can you update "bind it to an API/Query/Widget"

File: core-concepts/building-the-ui/README.md [hunk 2]
```diff
@@ -32,16 +32,16 @@ Appsmith has a collection of widgets that can be used to build the UI. Widgets c
...
 
 {% hint style="success" %}
-Widget names are important when adding logic so name your widgets well!
+Widget names are important while adding logic, so name your widgets well!
 {% endhint %}
 
 ## Editing widget properties 
 
-Widget properties can be editing via the property pane which can be toggled next to the delete icon. Widget properties can affect data, style and action to take. 
+Widget properties can be edited via the property pane which could be opened using top-right icon (Edit Widget Properties). Widget properties can affect the widget data, styling and actions.
```
Comment (Nikhil-Nandagopal) [inline core-concepts/building-the-ui/README.md:40]: instead of could, let's use "is"

**Review by Nikhil-Nandagopal — CHANGES_REQUESTED** [review]
@Saket2 had a few ideas here, the rest looks good

**Comment by Saket2** [comment]
@Nikhil-Nandagopal Made the suggested changes.

**Review by Nikhil-Nandagopal — APPROVED** [review]

**Pull request merged**